### DISCUSSION TITLE
Will a digitally signed wimboot binary be made available?

### DISCUSSION THREAD

**Discussion by oschroeder8** [ipxe/wimboot#14] [Q&A] [answered]
A signed wimboot binary of version 2.6.0 used to be available for download and actually worked with Secure Boot. Now that new releases are available, is there any chance that an updated, signed wimboot binary will be made available? I would love to be able to take advantage of the new features but not lose the ability to secure boot.

**Comment by mcb30** [comment, marked as answer]
All the wimboot releases since v2.5.1 have Secure Boot signed binaries published via https://github.com/ipxe/wimboot/releases (and also checked in to the source tree on the corresponding `v2.x.y-signed` branches).

As per https://github.com/ipxe/wimboot/blob/master/RELEASE.md, releases will be marked as "prerelease" on GitHub until the Secure Boot signed binaries are available, at which point it becomes a full release.

You can download Secure Boot signed binaries for the latest v2.7.1 release from https://github.com/ipxe/wimboot/releases/tag/v2.7.1

**Reply by oschroeder8** [reply]
Awesome, thank you!!
